### DISCUSSION TITLE
Add support for git creation date localized

### DISCUSSION THREAD

**Discussion by timvink** [squidfunk/mkdocs-material#2577] [Ideas]
## Description

Hi @squidfunk! I just released [mkdocs-git-revision-date-localized-plugin v0.9](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/releases/tag/v0.9), which adds a new tag `page.meta.git_creation_date_localized` (implemented by @PegasisForever).

I'd like to offer native support for that tag as well in mkdocs-material. It's backwards compatible; if a user specifies `enable_creation_date: true`, the creation dates should also show up.

Implementation-wise, that means:

- Updating the `source-date.html` partial with an additional if/else
- Updating all translation files, with a new label ('Created: '), with some basic translations filled by using google translate.
- Remove instructions on how to setup the partial override for mkdocs-material in our own docs

Would you be open to a PR for this?

### Use Cases

- See page creation date.

### Screenshots / Mockups

![image](https://user-images.githubusercontent.com/5570380/114404674-36ba7d80-9ba6-11eb-8c41-aa68e0022915.png)

**Comment by squidfunk** [comment]
Thanks for suggesting. Yes, you can open a PR and add support!

**Comment by squidfunk** [comment]
Moving to discussions, since this issue was essentially a question.
